pycallnumber/utils.py

```diff
@@ -149,15 +160,29 @@ def get_terminal_size(default_width=100, default_height=50):
         fmt = 'HHHH'
 
     winsize_struct = struct.pack(fmt, 0, 0, 0, 0)
-    try:
-        packed_winsize = fcntl.ioctl(0, termios.TIOCGWINSZ, winsize_struct)
-    except IOError:
-        height, width = (default_height, default_width)
-    else:
-        height, width, _, _ = struct.unpack(fmt, packed_winsize)
+    packed_winsize = fcntl.ioctl(0, termios.TIOCGWINSZ, winsize_struct)
+    height, width, _, _ = struct.unpack(fmt, packed_winsize)
     return width, height
 
 
+def _get_terminal_size_windows():
+    from ctypes import windll, create_string_buffer
+    # stdin handle is -10
+    # stdout handle is -11
+    # stderr handle is -12
+    handle = windll.kernel32.GetStdHandle(-12)
+    csbi = create_string_buffer(22)
+    res = windll.kernel32.GetConsoleScreenBufferInfo(handle, csbi)
+    # Windows and Unix-like systems have different ways of going about finding terminal size.
+    # Throw an IOError on failure to stay similar to the Unix-like API (ioctl throws IOError on failure).
+    if res:
+        _, _, _, _, _, left, top, right, bottom, _, _ = struct.unpack("hhhhHhhhhhh", csbi.raw)
+        width = right - left + 1
+        height = bottom - top + 1
+        return width, height
+    raise IOError
+
```

Comment:
The rest of `pycallnumber` follows an 80-character line-width convention. Lines 176, 177, and 179 all exceed that, causing flake8 tests to fail. I know this is an incredibly small, nit-picky, and even somewhat arbitrary detail.